My-Portfolio/index.html

```diff
@@ -8,572 +8,612 @@
+          <div class="flex-container" id="title">
+            <h2>My Recent works</h2>
+            <div id="workPlaceholder"></div>
+          </div>
+          <div class="flex-container" id="multi-post">
+            <div class="placeholder"></div>
+            <div class="flex-container" id="multi-postBlock">
+              <h3 id="titlePost">Multi-Post Stories</h3>
+              <p id="mpSupportText">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever
+                since the 1500s, when an unknown printer took a standard dummy text.
+              </p>
+              <ul class="flex-container" id="tagGroup">
+                <li class="tag">
+                  <a class="tagText" id="css">css</a>
+                </li>
+                <li class="tag">
+                  <a class="tagText" id="html">html</a>
+                </li>
+                <li class="tag">
+                  <a class="tagText" id="bootstrap">bootstrap</a>
+                </li>
+                <li class="tag">
+                  <a class="tagText" id="Ruby">Ruby</a>
+                </li>
+              </ul>
+              <div>
+                <button data-name="multiPost" class="button projectButton" type="button">
+                  <span id="buttonText">See Project</span>
+                </button>
+              </div>
+            </div>
+          </div>
+          <div class="grid-container" id="recent-works">
+            <div class="flex-container" id="cardWork">
+              <ul class="flex-container" id="cardElements">
+                <li>
+                  <h3 id="cardTitle">Profesional Art Printing Data</h3>
+                </li>
+                <li>
+                  <p id="cardDesc">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                  </p>
+                </li>
+                <li class="flex-container" id="cardTags">
+                  <ul class="flex-container" id="tagGroup">
+                    <li class="tag">
+                      <a class="tagText" id="html">html</a>
+                    </li>
+                    <li class="tag">
+                      <a class="tagText" id="bootstrap">bootstrap</a>
+                    </li>
+                    <li class="tag">
+                      <a class="tagText" id="Ruby">Ruby</a>
+                    </li>
+                  </ul>
+                </li>
+              </ul>
+              <div id="cardButton">
+                <button data-name="card1" class="button projectButton" type="button">
+                  <span id="buttonText">See Project</span>
+                </button>
               </div>
-              <a href="#" id="menu-button"></a>
-              <ul class="flex-container" id="deskNavBar">
+            </div>
```

Comment:
- [ ] As recommended in the project requirements [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.%20Remember%2C%20all%20of%20the%20HTML%20in%20that%20section%20is%20created%20when%20the%20page%20loads.) Kindly make sure the works section is generated dynamically with javascript by looping inside the array of data you have created :recycle: